### PR TITLE
Added support for prometheus metrics

### PR DESCRIPTION
**Pull request by harnas-google**

### CODE REVIEW

**Review by axtnsx — CHANGES_REQUESTED** [review]
Please, remove binary files (nginx-prometheus-exporter) from repository.

File: templates/docker-entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    if [[ $? -ne 0 ]]; then
+        sleep 1
+        tries=$((tries-1))
+    else
+        break
+    fi
+done
+set -e
+
+/usr/local/bin/nginx-prometheus-exporter
```
Comment (axtnsx) [inline templates/docker-entrypoint.sh]: nginx should be the main process on the container.

File: 1/debian9/1.15/docker-entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    if [[ $? -ne 0 ]]; then
+        sleep 1
+        tries=$((tries-1))
+    else
+        break
+    fi
+done
+set -e
+
+/usr/local/bin/nginx-prometheus-exporter
```
Comment (axtnsx) [inline 1/debian9/1.15/docker-entrypoint.sh]: nginx should be the main process on the container.

File: 1/debian9/1.14/docker-entrypoint.sh [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    if [[ $? -ne 0 ]]; then
+        sleep 1
+        tries=$((tries-1))
+    else
+        break
+    fi
+done
+set -e
+
+/usr/local/bin/nginx-prometheus-exporter
```
Comment (axtnsx) [inline 1/debian9/1.14/docker-entrypoint.sh]: nginx should be the main process on the container.

**Comment by harnas-google** [comment]
> Please, remove binary files (nginx-prometheus-exporter) from repository.

done

**Pull request closed without merging**